src/components/AddStudent.js

```diff
@@ -0,0 +1,67 @@
+import * as React from "react";
+import Button from '@mui/material/Button';
+import TextField from '@mui/material/TextField';
+ 
+class AddStudent extends React.Component {
+  constructor(props) {
```

Comment:
If there's no props being passed to this component, I believe you don't actually need to have the props variable passed to the constructor.

Reply:
This is correct, cleaned it up.